com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/security/AuthorizationChecker.java

```diff
@@ -111,11 +115,17 @@ boolean checkPermissions(UserInfo userInfo, String spaceGuid, boolean readOnly)
     }
 
     private boolean hasPermissions(CloudControllerClientSupportingCustomUserIds client, String userId, UUID spaceGuid, boolean readOnly) {
-
-        if (client.getSpaceDeveloperIdsAsStrings(spaceGuid)
+        // Get the space developers for a space GUID with cache.
+        if (SPACE_DEVELOPERS_CACHE.get(spaceGuid, (() -> getSpaceDevelopers(client, spaceGuid)))

```

Comment:
extract if expressions to previous statements or method calls

Reply:
Done.